Sources/tart/VMDirectory.swift

```diff
@@ -161,6 +181,175 @@ struct VMDirectory: Prunable {
     try diskFileHandle.close()
   }
 
+  private func resizeASIFDisk(_ sizeGB: UInt16) throws {
```

Comment:
It would be logical to move this function and functions below to `DiskImageFormat` instead now that we have it.